vcluster/configure/vcluster-yaml/sleep-mode.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: Sleep Mode
+sidebar_label: sleepMode
+sidebar_position: 7
+sidebar_class_name: pro
+description: Learn how to configure and manage sleep mode functionality in virtual clusters without using an agent.
+---
+import ProAdmonition from '../../_partials/admonitions/pro-admonition.mdx'
+import SleepMode from '../../_partials/config/sleepMode.mdx'
+import SleepModeDeploymentExample from '../../../vcluster/_fragments/sleepmode-deployment-example.mdx'
+import SleepModeIngressExample from '../../../vcluster/_fragments/sleepmode-ingress-example.mdx'
+import BasePrerequisites from '../../../docs/_partials/base-prerequisites.mdx';
+import InstallCli from '../../_partials/deploy/install-cli.mdx';
+
+<ProAdmonition />
+
+# Sleep mode
+
+Not all workloads need to run all the time, and scaling them down saves time and money. With sleep mode you can scale down workloads based on a set schedule, or activity from users and ingress.
+
+:::warning
+Sleep mode is intended for pre-production use cases only, and comes with some limitations and
+caveats when used on a stand alone vCluster that is not connected to [the Platform](../../../../platform/use-platform/virtual-clusters/key-features/sleep-mode).
+:::
+
+## Enable sleep mode
+
+To enable sleep mode, use the following configuration inside your `vcluster.yaml`:
+
+```yaml title="sleep mode configuration"
+sleepMode:
+  enabled: true
+  autoSleep:
+    afterInactivity: 1h
+```
+
+## How it works
+
+### Sleep mode operations
+
+Sleep mode performs two main operations:
+
+**Sleeping**
+Deletes bare pods and scales down the following resources:
+- Deployments
+- ReplicaSets
+- ReplicationControllers
+- DaemonSets
+
+**Waking**
+
+- Scales resources back to their original state
+- Cannot restore previously deleted bare pods
+
+### Resource exemption
+
+Resources can be exempted from sleep mode using either:
```

Comment:
Nit: remove either as it's more than 2 things:
```suggestion
Resources can be exempted from sleep mode by:
```
Could also say "any of the following" or something